src/wp-content/themes/twentytwentyone/assets/css/ie-editor.css

```diff
@@ -1657,7 +1657,7 @@ pre.wp-block-preformatted {
 }
 
 .wp-block-quote .wp-block-quote__citation {
-	color: #28303d;
+	color: currentColor;
```

Comment:
Not sure why this changed but it should have no effect as we don't support IE11 as of WP 5.8